app/services/data/auto-approve-claim.js

```diff
@@ -1,15 +1,19 @@
 const config = require('../../../knexfile').asyncworker
 const knex = require('knex')(config)
 
+const autoApproveClaimExpenses = require('./auto-approve-claim-expenses')
 const insertTask = require('../data/insert-task')
 const tasksEnum = require('../../constants/tasks-enum')
 const statusEnum = require('../../constants/status-enum')
 
-module.exports = function (claim) {
+module.exports = function (claimAndAutoApprovalData, visitorEmailAddress) {
```

Comment:
only take ClaimId, visitorEmailAddress. Makes simpler to test and avoids problems if claimAndAutoApprovalData changes